README.md

```diff
@@ -35,26 +34,31 @@ nvm install
 nvm use
 ```
 
+### Upgrade npm
+
+Installing Cumulus requires npm version 5.7.0 or later. You can upgrade your
+version of npm by running `npm install npm@latest -g`
+
 ### Install Lerna
 
 We use Lerna to manage multiple Cumulus packages in the same repo. You need to install lerna as a global module first:
 
-    $ yarn global add lerna
+    $ npm install -g lerna
 
 ### Install Local Dependencies
 
-We use yarn for local package management
+We use npm for local package management
 
-    $ yarn install
-    $ yarn ybootstrap
+    $ npm ci
```

Comment:
Why would we use `npm ci` for local development?

Reply:
I know that `npm ci` is much more deterministic based on the lockfile, I'm just not used to seeing it used for local development. But I can't really think of a reason why it shouldn't be.